src/types/types.tsx

```diff
@@ -1,3 +1,21 @@
+export type Indexable = {
+    [key: string]: any
+}
+
+export type selectionType = Indexable & {
+    FLOW: null | string,
```

Comment:
In the DB i originally made flow required, since our original app was based on menstruation. Do you think I should change that, or should we change the type? (also, we would have to figure out how to make that required in our form, which seems challenging given that theyre just buttons?

Reply:
I think we should make it required! I can easily add some form validation to make sure that a flow type is selected before the form can be submitted